warehouse/predicates.py

```diff
@@ -96,6 +91,11 @@ def __call__(self, context: Organization | Team, request):
             )
         ):
             return True
+        elif (
+            # Organization accounts are disabled.
+            request.flags.enabled(AdminFlagValue.DISABLE_ORGANIZATIONS)
+        ):
+            return False
```

Comment:
Shouldn't this still come first?

Reply:
No, we're not flipping the flag until we're ready for open signup.

Reply:
Same logic as is in https://github.com/pypi/warehouse/pull/13392/files#diff-fd646332e15b1650ac3a05f944bf6ae36335637b687d06676bb67f7d6f3b1fefR70-R79

This allows users who have an organization to bypass the flag

Reply:
Got it!